modules/programs/scmpuff.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.programs.scmpuff;
+in {
+  meta.maintainers = [ maintainers.cpcloud ];
+
+  options.programs.scmpuff = {
+    enable = mkEnableOption
+      "Work with git from the command line quicker, by substituting numeric shortcuts for files.";
```

`mkEnableOption` expands the description to `Whether to enable ${x}.`, where `x` is the string you passed as argument.

Also note that the final period is already present.

```suggestion
    enable = mkEnableOption
      "scmpuff, a command line tool that allows you to work quicker with Git by substituting numeric shortcuts for files";
```